development_workflow.md

```diff
@@ -446,61 +456,67 @@ collaborator.
 
 Now all those people can do:
 
-    git clone git@github.com:your-user-name/nest-simulator.git
+```bash
+git clone git@github.com:your-user-name/nest-simulator.git
+```
 
 Remember that links starting with `git@` use the ssh protocol and are
 read-write; links starting with `git://` are read-only.
 
 Your collaborators can then commit directly into that repo with the
 usual:
 
-     git commit -am 'ENH - much better code'
-     git push origin master # pushes directly into your repo
+```bash
+git commit -am 'ENH - much better code'
+git push origin master # pushes directly into your repo
+```
 
 ----------------------------------------------------------------------------
 
-Use cases for the new platform
```

Comment:
I think we should remove these use cases altogether. They were useful to convince everyone that the new platform can do everything we need, but now the workflow descriptions are much better suited for real work. Moreover, we never managed to formulate all of them into proper sentences.